client-java/controller/src/main/java/org/evomaster/client/java/controller/SutHandler.java

```diff
@@ -79,6 +79,67 @@ default void setupForGeneratedTest(){}
      * Execute the given data insertions into the database (if any)
      *
      * @param insertions DTOs for each insertion to execute
+     * @param previous an array of insertion results which were executed before this execution
+     * @return insertion execution results
      */
     InsertionResultsDto execInsertionsIntoDatabase(List<InsertionDto> insertions, InsertionResultsDto... previous);
+
+
+    /**
+     * <p>
+     * return an instance of a client of an RPC service.
+     * </p>
+     *
+     * <p>
+     * This method must be blocking until the SUT is initialized.
+     * </p>
+     *
+     * <p>
+     * This method is only required when the problem is RPC for the moment,
+     * otherwise return null
+     * </p>
+     *
+     * TOCHECK with Andrea if it is a proper place where we put this method
+     * this method is required
```

Comment:
aslo, why returning an `Object`, instead of something like:
`default <T,T extends X> T getRPCClient(Class<X> x)`? 
(not sure on the syntax here on GitHub editor)

Reply:
each interface could have a corresponding type for its client,  (it might not always be true, as T extends X or X extends T, see native thrift client). Also in the driver, the RPC problem is specified with `Map<String, Object> mapOfInterfaceAndClient` for interfaces to client instances. then here we return `Object`

Reply:
we could pass a `Class<?>` instead of `String`, before I thought that there might have dependency or class loader problem. is it ok with `string`, then use `Class.forName` to get class with string?

Reply:
hmm... i see. i guess it is ok for now

Reply:
i guess this will become more clear when we are going to generate tests using the interfaces directly